src/realm/sort_descriptor.cpp

```diff
@@ -591,3 +606,47 @@ void DescriptorOrdering::get_versions(const Group* group, TableVersions& version
         versions.emplace_back(table_key, group->get_table(table_key)->get_content_version());
     }
 }
+
+std::string SemanticSearchDescriptor::get_description(ConstTableRef) const
+{
+    return "KNN()";
```

Comment:
We should fill this in if you'd like to eventually have support for this feature in the query parser. Is that something you'd like me to add?